packages/wonder-blocks-modal/components/scroll-disabler.js

```diff
@@ -17,14 +17,18 @@ const needsHackyMobileSafariScrollDisabler = (() => {
     return userAgent.indexOf("iPad") > -1 || userAgent.indexOf("iPhone") > -1;
 })();
 
-class ScrollDisabler extends Component<{||}> {
+type Props = {||};
+
+class ScrollDisabler extends Component<Props> {
     static oldOverflow: string;
     static oldPosition: string;
     static oldScrollY: number;
     static oldWidth: string;
     static oldTop: string;
 
-    componentWillMount() {
+    constructor(props: Props) {
```

Comment:
I don't think this change is equivalent (and I also think it wasn't right before). The `constructor` shouldn't have side effects so I think this actually should become `componentDidMount` instead.

Reply:
Will do.  I was thinking about side effects in terms of changes the component was making to things outside of itself instead of things outside of itself that the component was using. 😅